pkg/kubeadm/tlsutil/tlsutil.go

```diff
@@ -0,0 +1,133 @@
+// Copied from https://github.com/coreos/bootkube/blob/master/pkg/tlsutil/tlsutil.go
+// TODO vendor it properly
+// TODO also see whether some of these should just go into "pkg/util/certificates"
```

Comment:
Voting for making a super-usable `pkg/util/crypto` package with subpackages if required.
At some point in the future, we may consider a such all-around cert management package could be split out of main repo.